panacea/lib/panacea/pml/analysis/unnamed.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Panacea.Pml.Analysis.Unnamed do
+
+  @composite [:task, :sequence, :branch, :selection, :iteration]
+
+  def run(ast) do
+    analyse([], ast)
+  end
+
+  defp analyse(result, {type, attrs, children}) when type in @composite do
+    result =
+      if !Keyword.has_key?(attrs, :name) do
+        [%{type: type, line: Keyword.get(attrs, :line)} | result]
+      else
+        result
+      end
+
+    Enum.reduce(children, result, fn (child, acc) ->
+      analyse(acc, child)
+    end)
+  end
+  defp analyse(result, {_, _, children}) when is_list(children) do
```

Comment:
Yeah, we still want to process them. If I didn't have this case they would never be processed further